pykusto/expressions.py

```diff
@@ -96,6 +96,13 @@ def has(self, exp: StringType) -> 'BooleanExpression':
         # The pattern for the search expression must be a constant string.
         return BooleanExpression(KQL(f'{self.kql} has {to_kql(exp)}'))
 
+    def has_cs(self, exp: StringType) -> 'BooleanExpression':
+        """
+        https://docs.microsoft.com/en-us/azure/data-explorer/kusto/query/datatypes-string-operators
+        """
+        # The pattern for the search expression must be a constant string.
```

Comment:
I don't understand this comment. Do you mean it has to be a **literal** string? If so, change the input type from `StringType` to `str`

Reply:
agree, it was missed in a previous PR when I moved it from functions to expressions.